File: tests/fixtures/sample_repo/swift_pkg/Calculator.swift
import Foundation

/// Calculator for basic arithmetic.
class Calculator: Computable {
    func add(x: Double, y: Double) -> Double {
        return x + y
    }

    func subtract(x: Double, y: Double) -> Double {
        return x - y
    }

    private func helper() {}

    func compute() -> Double {
        return 0.0
    }
}

File: tests/fixtures/sample_repo/csharp_pkg/Calculator.cs
using System;
using Sample.Models;

namespace Sample
{
    /// <summary>Calculator for basic arithmetic.</summary>
    public class Calculator
    {
        public double Add(double x, double y)
        {
            return x + y;
        }

        public double Subtract(double x, double y)
        {
            return x - y;
        }

        private void Helper() { }
    }
}
